its/src/test/java/com/sonar/it/shared/TestUtils.java

```diff
@@ -160,6 +160,10 @@ public static String replaceLtsVersion(String version) {
     return version;
   }
 
+  public static boolean hasModules(Orchestrator orch) {
+    return !orch.getServer().version().isGreaterThanOrEquals(7, 6);
```

Comment:
[minor, cosmetic] this would have been easier to read if it was `orch.getServer().version().lessThan(7, 6)` (and without `!`)

Reply:
If there is such method of course :)

Reply:
There is not obviously :D